core/scss/components/date-range-picker/module.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../tools/functions/core';
+
+@import '../../colors/settings';
```

Comment:
Doesn't seem all these imports are needed - please confirm.

Reply:
Just refer to date-picker. Because use date-picker inside.